README.md

```diff
@@ -1,8 +1,25 @@
 # demUtils
-Demographics Related Utility Functions
 
 <!-- badges: start -->
-[![Travis build status](https://travis-ci.org/ihmeuw/demUtils.svg?branch=master)](https://travis-ci.org/ihmeuw/demUtils)
+[![R build status](https://github.com/ihmeuw/demUtils/workflows/R-CMD-check/badge.svg)](https://github.com/ihmeuw/demUtils/actions)
```

Comment:
Can you add info about this to the Hub page?

Reply:
I'm not exactly sure how it works. Copied over the formatting from the tidyverse repos and they seemed to work